ALapina/grid_game
# Add build config

Adds Rollup configuration to build the final js file + adds dev server

## How to test changes
1. `npm install` to install dependencies
1. `npm run dev` -> should open the browser tab with the game
1. `npm run build` -> will create a final build file, it should be ~ 12K
